styles/content/typography/body/styles.less

```diff
@@ -2,45 +2,40 @@
   /* body */
 
   html,
-  body,
-  p {
+  body {
     .element-text-size(body, null);
     .element-text-style(body);
     display: block;
 
     & when (@body-emphasis-enabled) {
 
-      &.emphasis,
-      .emphasis {
+      span.emphasis {
```

Comment:
I'm in favor of it too.  What I'm asking is, in doing so, will I potentially break anything in DC/OS today?